src/server.rs

```diff
@@ -58,7 +58,29 @@ pub fn run<S: Storage>(c: Config, s: S) {
         .serve(new_service)
         .map_err(|e| error!("server error: {}", e));
     info!("Listening on {}", addr);
-    hyper::rt::run(server);
+    let mut builder = tokio::runtime::Builder::new();
+    if let Some(num) = get_core_threads() {
+        log::info!("Set core_threads to {}", num);
+        builder.core_threads(num);
+    }
+    let mut entered = tokio_executor::enter().expect("nested tokio::run");
```

Comment:
Most of these parts are copy-pasted from `hyper::rt::run()`
https://docs.rs/hyper/0.12.31/src/hyper/rt.rs.html#41-46
https://docs.rs/tokio/0.1.21/src/tokio/runtime/threadpool/mod.rs.html#101-111